analysers/analyser_osmosis_highway_deadend.py

```diff
@@ -442,8 +446,16 @@ def __init__(self, config, logger = None):
             resource = 'https://wiki.openstreetmap.org/wiki/Tag:service%3Ddrive-through')
 
         self.callback21 = lambda res: {"class": 1, "data": [self.way_full, self.node_full, self.positionAsText]}
-        self.callback22 = lambda res: {"class": 2, "data": [self.way_full, self.node_full, self.positionAsText]}
-
+        self.callback22 = lambda res: {
+            "class": 2,
+            "data": [self.way_full, self.node_full, self.positionAsText],
+            "text": {"en": "Unconnected highway: {}".format(res[3])}
```

Comment:
As "Unconnected highway" is already the label of the class. You just set the highway value.
```suggestion
            "text": {"en": res[3]}
```

Reply:
Oh, this was actually my suggestion 🙈. It makes the list view more readable (because nothing is wrong with the highway value itself)

Reply:
@frodrigo just tagging to make sure you saw the comment as it was applied immediately.

Now there will be a strange text "service" or so, while there's nothing wrong with the highway=service tag

Reply:
Too much notif, I do not read it.

As opposite, the text should use translation ( with `T_()`). And be translated by translator, removing the text was to avoid translation work. `text` is also know as "subtitle", and should come with the title.

Reply:
Hmm, ok. Still not my preference for the list view as it doesn't give information about the actual issue.